api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxPvlsDataset.java

```diff
@@ -64,51 +64,40 @@ public DataSetDefinition constructTxPvlsDatset() {
-		// months and match location", EptsReportUtils.map(
-		// hivIndicators.getPatientsOnArtForMoreThan3MonthsAndMatchLocation(),
-		// "endDate=${endDate},location=${location}"), "");
-		// add breastfeeding and pregnant Numerator
-		
 		// add patients on routine for adults and children
 		dsd.addColumn("B01", "Breast feeding and on routine Numerator", EptsReportUtils.map(
-		    breastfeedingIndicators.getBreastfeedingWomenWithSuppressedViralLoadIn12MonthsOnRoutineNumerator(), mappings),
+		    eptsGeneralIndicator.getIndicator("breastfeedingWomenWithViralLoadSuppression routine numerator", EptsReportUtils.map(txPvls.getBreastFeedingWomenOnRoutineNumerator(), mappings) ), mappings),
 		    "");
 		dsd.addColumn("B02", "Breast feeding and NOT documented Numerator",
 		    EptsReportUtils.map(
-		        breastfeedingIndicators.getBreastfeedingWomenWithSuppressedViralLoadIn12MonthsNotDocumentedNumerator(),
-		        mappings), "");
+		        eptsGeneralIndicator.getIndicator("breastfeedingWomenWithViralLoadSuppression not documented numerator", EptsReportUtils.map(txPvls.getBreastFeedingWomenNotDocumentedNumerator(), mappings) ), mappings),
+				"");
 		dsd.addColumn("B03", "Pregnant and on routine Numerator", EptsReportUtils.map(
-		    pregnantIndicators.getPregnantWomenWithSuppressedViralLoadIn12MonthsAndOnRoutineNumerator(), mappings), "");
+				eptsGeneralIndicator.getIndicator("pregnantWomenWithViralLoadSuppression and on routine", EptsReportUtils.map(txPvls.getPregnantAndOnRoutineNumerator(), mappings) ), mappings),
+				"");
 		dsd.addColumn("B04", "Pregnant and NOT documented Numerator", EptsReportUtils.map(
-		    pregnantIndicators.getPregnantWomenWithSuppressedViralLoadIn12MonthsNotDocumentedNumerator(), mappings), "");
+				eptsGeneralIndicator.getIndicator("pregnantWomenWithViralLoadSuppression and NOT documented", EptsReportUtils.map(txPvls.getPregnantAndNotDocumentedNumerator(), mappings) ), mappings),
+				"");
 		
 		// add breastfeeding and pregnant Denominator
 		dsd.addColumn(
 		    "B05",
 		    "Breast feeding and on routine Denominator",
 		    EptsReportUtils.map(
-		        breastfeedingIndicators.getBreastfeedingWomenWithViralLoadIn12MonthsAndOnRoutineDenominator(), mappings), "");
+		        eptsGeneralIndicator.getIndicator("breastfeedingWomenWithViralLoad-routine denominator", EptsReportUtils.map(txPvls.getBreastfeedingWomenOnRoutineWithViralLoadResultsDenominator(), mappings) ), mappings),
+				"");
 		dsd.addColumn(
 		    "B06",
 		    "Breast feeding and NOT documented Denominator",
 		    EptsReportUtils.map(
-		        breastfeedingIndicators.getBreastfeedingWomenWithViralLoadIn12MonthsAndNotDocumentedDenominator(), mappings),
-		    "");
+		        eptsGeneralIndicator.getIndicator("breastfeedingWomenWithViralLoad-not documented denominator", EptsReportUtils.map(txPvls.getBreastfeedingWomenAndNotDocumentedWithViralLoadResultsDenominator(), mappings) ), mappings),
```

Comment:
The second mappings parameter is not needed here. Same applies a few times in this file.